apis/beacon/states/validators.yaml

```diff
@@ -0,0 +1,47 @@
+get:
+  operationId: "getStateValidators"
+  summary: "Get validators from state"
+  description: "Returns filterable list of validators with their balance, status and index."
+  tags:
+    - Beacon
+  parameters:
+    - name: state_id
+      $ref: '../../../beacon-node-oapi.yaml#/components/parameters/StateId'
+    - name: id
+      description: "Either public key or validator index"
```

Comment:
And what happened to the duck typing discussion for query params? Is it ok to do it like this?

Reply:
Seems like nobody complained about that but I can separate it so we avoid duck typing here